queue_services/entity-emailer/flags.json

```diff
@@ -0,0 +1,5 @@
+{
+    "flagValues": {
+        "enable-specific-service-provider": true
+    }
+}
```

Comment:
As per our conversation, I think it makes sense to name the flag `disable-specific-service-provider` with a default value of `false`.  When we toggle the flag to stop serving the Dye & Durham text, this flag will toggle to `true`.